rootfs/scripts/utils.sh

```diff
@@ -75,40 +75,45 @@ wait_for_lock() {
 	update_branch $__branch
 
 	# if we are not the first in line, spin
-	if [ "$(cat $__mutex_queue_file | head -n 1)" != "$__ticket_id" ]; then
+	if [ "$(cat $__queue_file | awk NF | head -n 1)" != "$__ticket_id" ]; then
 		sleep 5
 		wait_for_lock $@
 	fi
 }
 
-# Wait for the lock to become available
+# Remove from the queue, when locked by it or just enqueued
 # args:
 #   $1: branch
 #   $2: queue_file
 #   $3: ticket_id
-unlock() {
+unqueue() {
```

Comment:
Can we rename this to `dequeue`? I think that's the more common word for this action